src/Pages/AdminPage/Store.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import './Store.scss';
+
+const Store = ({ text }) => {
+  console.log(text);
```

Comment:
```suggestion
const Store = ({ text }) => {
// 구조 분해 할당
  console.log(text);
```

text를 구조분해 할당해서 사용해주세요!